### ISSUE TITLE
Support for new architecture

### ISSUE THREAD

**Issue by niki-bri** [SHISME/react-native-draggable-grid#96]
Will this library work on the new architecture? 
I am currently testing it and it is quite buggy.

**Comment by yrjwcharm** [comment]
@niki-bri  this library's unable to work

**Comment by niki-bri** [comment]
Yes, I am quite aware of that. My question is if there are any plans for supporting the new architecture. Thx
